_stylesheet.scss

```diff
@@ -261,6 +262,10 @@ $dock_style_modes: [null, shrink, extended, extended-shrink];
     background-color: $remark_color;
 }
 
+.dash-item-container .overview-tile .app-grid-running-dot {
+    margin-bottom: $dash_padding_gnome - 3px;
```

Comment:
Changing 3px to 4px resolves my concern about the slightly insufficient spacing. With 4px it now looks the same as upstream gnome-shell. Would 4px work for you?

Reply:
No problem. I'll do the change.

Reply:
Done.

Reply:
Thanks. The change should have been squashed into the first commit so I force-pushed the squash into your branch. I will now merge it...